### DISCUSSION TITLE
Where is the binary file for v1.11.2？

### DISCUSSION THREAD

**Discussion by zhangsj** [filecoin-project/lotus#7319] [Storage Provider] [answered]
Where is the binary file for v1.11.2？

**Comment by de-authority** [comment, marked as answer]
I think it's not an issue. and converte to discussion.
we don't supply binary file for v1.11.2 now. 
you can compile by your own. and you may need this docs: https://docs.filecoin.io/get-started/lotus/installation/#building-from-source
